gspread/client.py

```diff
@@ -11,7 +11,12 @@
 import re
 import warnings
 
-from xml.etree import ElementTree
+try:
+    from lxml import etree
+    fromstring = etree.fromstring # Roughly 3x faster parsing for large worksheets
+except ImportError:
+    from xml.etree import ElementTree
+    fromstring = ElementTree.fromstring
```

Comment:
I think both `try` and `except` can import `ElementTree` into the namespace:

~~~python
try:
    from lxml import etree as ElementTree
except ImportError:
    from xml.etree import ElementTree
~~~

This way the change will be even simpler.